suites/squid/common/regression/ss_rgw_ssl.yaml

```diff
@@ -0,0 +1,398 @@
+      config:
+        script-name: test_Mbuckets_with_Nobjects.py
+        config-file-name: test_user_stats_reset.yaml
+
+  # kafka broker type broker with persistent flag enabled
+  - test:
+      name: notify put,delete events with kafka_broker_persistent with rgw ssl
+      desc: notify put,delete events with kafka_broker_persistent with rgw ssl
+      module: sanity_rgw.py
+      polarion-id: CEPH-83574489
+      config:
+        run-on-rgw: true
+        extra-pkgs:
+          - wget https://download.oracle.com/java/23/latest/jdk-23_linux-x64_bin.rpm
+        install_start_kafka: true
+        script-name: test_bucket_notifications.py
+        config-file-name: test_bucket_notification_kafka_broker_persistent_delete.yaml
+
+  - test:
+      abort-on-fail: true
+      config:
+        install:
+          - agent
+      desc: Setup and configure vault agent
+      destroy-cluster: false
+      module: install_vault.py
+      name: configure vault agent
+      polarion-id: CEPH-83575226
+
+  - test:
+      abort-on-fail: false
+      config:
+        suite:
+          - JavaS3
+        kms_keyid: testKey01
+        ssl: true
+      desc: Java S3tests
+      destroy-cluster: false
+      module: java_s3tests.py
+      name: execute Java s3tests
+      polarion-id: CEPH-83586289
+
+  - test:
+      abort-on-fail: false
+      config:
+        suite:
+          - maven
+        ssl: true
+      desc: Java Maven Test suite
+      destroy-cluster: false
+      module: java_s3tests.py
+      name: execute Java Maven test suite
+      polarion-id: CEPH-83586289
+
+  - test:
+      abort-on-fail: false
+      config:
+        branch: ceph-squid
+        kms_keyid: testKey01
+      desc: S3tests
+      comments: Known issue BZ-2312112 targeted to 8.0z2
+      destroy-cluster: false
+      module: test_s3.py
+      name: execute s3tests
+      polarion-id: CEPH-83575225
+
+  - test:
+      name: object level retention test Governance mode
+      desc: object level retention test Governance mode
+      polarion-id: CEPH-83574058
+      module: sanity_rgw.py
+      config:
+        script-name: test_object_level_retention.py
+        config-file-name: test_object_level_governance.yaml
+
+  - test:
+      name: Test the byte ranges with get object
+      desc: Test the byte ranges with get_object
+      polarion-id: CEPH-83572691
+      module: sanity_rgw.py
+      config:
+        test-version: v2
+        script-name: test_byte_range.py
+        config-file-name: test_byte_range.yaml
+
+  - test:
+      name: check-ceph-health
+      module: exec.py
+      config:
+        cephadm: true
+        commands:
+          - "ceph -s"
+      desc: Check for ceph health debug info
+      polarion-id: CEPH-83575200
+
+  - test:
+      name: remove rgw ssl cert daemon from cluster
+      desc: remove rgw ssl cert daemon from cluster
+      module: exec.py
+      config:
+        cephadm: true
+        commands:
+          - "ceph orch rm {service_name:.rgw.ssl}"
```

Comment:
If possible clear other resources created in between tests, like nfs mount

Reply:
sure @vamahaja will check